### PR TITLE
fix: packer checklist position

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksfix-check-item-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1202097197789424/1203368245666617/f) 🎟️

## 🗒️ What

Fixes a bug where packer checklist `::before` content wasn't rendering in the correct position. 

### Before
Screenshot 2023-07-07 at 4 14 37 PM

### After
Screenshot 2023-07-07 at 4 14 27 PM

## 🧪 Testing

- [Visit path](https://dev-portal-git-ksfix-check-item-hashicorp.vercel.app/packer/docs/partnerships#checklist) with checklist 
- Validate that it is positioned as expected.
- Lists outside of this packer checklist primitive should remain unaffected

### CODE REVIEW

File: src/components/author-primitives/packer/checklist/style.module.css [hunk 1]
```diff
@@ -6,13 +6,15 @@
...
+		margin-left: 8px;
 
 		& li {
-			color: #525252;
+			color: var(--token-color-foreground-primary);
+			position: relative;
 
 			&::before {
 				content: '';
+				position: absolute;
```
Comment (kendallstrautman) [inline src/components/author-primitives/packer/checklist/style.module.css:17]: Note this is all specific to packer only.

File: src/components/author-primitives/packer/checklist/style.module.css [hunk 2]
```diff
@@ -6,13 +6,15 @@
 .root {
 	& ul {
 		list-style: none;
-		margin-left: 2rem;
+		margin-left: 8px;
```
Comment (kendallstrautman) [inline src/components/author-primitives/packer/checklist/style.module.css:9]: @braydenlove let me know what spacing you'd like here. I tried to take reference from the regular list item margin.

**Review by braydenlove — COMMENTED** [review]
@kendallstrautman this is awesome, thanks for jumping on these old tickets!!! Can we do `top: 4px;` instead of `7px;`?
image

File: src/components/author-primitives/packer/checklist/style.module.css (hunk 2 above)
Comment (braydenlove) [reply src/components/author-primitives/packer/checklist/style.module.css:9]: I just commented, thanks again!

**Review by braydenlove — APPROVED** [review]
Thanks!

**Review by zchsh — APPROVED** [review]
✅ Looks great from a code perspective! 💯 And checked out the preview, looks good to me 👍

**Pull request merged**